src/api/java/mezz/jei/api/recipe/transfer/IRecipeTransferInfo.java

```diff
@@ -38,4 +38,11 @@
 	 * Return a list of slots that the transfer can use to get items for crafting, or place leftover items.
 	 */
 	List<Slot> getInventorySlots(C container);
+
```

Comment:
I used a default interface method to introduce the flag so that the change wouldn't break any existing implementations.